README.rst

```diff
@@ -1,81 +1,155 @@
 |
-| SMECS facilitates the extraction of research software metadata from repositories on GitHub/GitLab. It offers a user-friendly graphical user interface for visualizing the retrieved metadata. This empowers researchers to create good metadata for their research software without reentering data which is already available elsewhere. Ultimately, SMECS delivers the curated metadata in JSON format, enhancing usability and accessibility.
+| SMECS facilitates the extraction of research software metadata from GitHub and GitLab repositories. It provides a user-friendly graphical interface for visualizing the retrieved metadata, enabling researchers and research software engineers to create high-quality metadata without reentering information already available elsewhere. The curated metadata is exported as CodeMeta-compliant JSON, ensuring integration with other tools and enhancing the discoverability, reuse, and impact of research software.
 |
 |
 | **Authors:** Stephan Ferenz, Aida Jafarbigloo
 |
-Key Stages in SMECS
+Phases in SMECS
 __________________________________________________________
-| The figure below illustrates the sequential processes and data flows within SMECS. First, users input data, triggering the tool to extract metadata associated with specific URLs. This metadata is then visualized, allowing users to review and interact with it. Users can curate, modify, and finalize the metadata according to their needs. Once satisfied, they can download the curated metadata in JSON format, providing an interoperable output for further use.
+| The workflow of SMECS consists of four sequential phases: **Start**, **Extraction**, **Curation**, and **Export**.
 |
 |
-.. image:: https://github.com/NFDI4Energy/SMECS/blob/master/docs/diagram.png
-   :alt: SMECS Workflow Visualization
+
+.. image:: https://github.com/NFDI4Energy/SMECS/blob/master/docs/Extraction_via_hermes-1.png
+   :alt: SMECS Workflow
    :width: 1000px
+
+|
+|
+
+1. **Start Phase**
+__________________________________________________________
+In the Start phase, users provide two key inputs:
+      - A repository link (GitHub or GitLab)
+      - A personal access token for the corresponding platform
+SMECS can operate without user-provided tokens for some repositories by using internal default tokens. However:
+      - For other GitLab instances, a user-provided token is always required.
+      - Providing a token can enable SMECS to extract more detailed metadata from certain repositories.
+|
+2. **Extraction Phase**
+__________________________________________________________
+The Extraction phase uses `HERMES <https://github.com/softwarepub/hermes>`_ harvesting steps to retrieve metadata from multiple sources. For details on the specific metadata fields, see: `Metadata Terms in SMECS <https://github.com/NFDI4Energy/SMECS/blob/master/docs/metadata-terms.md>`_. Once the inputs from the Start phase are submitted, SMECS initiates metadata retrieval using four HERMES harvesters:
```

Comment:
I would suggest to remove that file and add a link to the README of the harvesters.